tide/master/MasterApplication.cpp

```diff
@@ -566,13 +566,25 @@ void MasterApplication::_initRestInterface()
                 _logger.get(), &LoggingUtility::logScreenStateChanged);
 
         connect(_planarController.get(), &PlanarController::powerStateChanged,
-                [this](ScreenState state) {
+                [this](const ScreenState state) {
                     if (state == ScreenState::ON)
                         _inactivityTimer->restart();
                     else
                         _inactivityTimer->stop();
                 });
 
+        connect(_planarController.get(), &PlanarController::powerStateChanged,
+                [this](const ScreenState state) {
+                    if (state == ScreenState::OFF)
+                        _lock->unlock();
+                });
+
+        connect(_planarController.get(), &PlanarController::powerStateChanged,
+                [this](const ScreenState state) {
+                    if (state == ScreenState::OFF)
+                        _lock->unlock();
+                });
+
```

Comment:
the second one is not really needed I guess

Reply:
done